Add alternate house image for specific screen sizes

maya opened the pull request:
This adds an alternate house image for specific screen sizes using an aspect-ratio media query.

TODO:
- [x] get a shorter house image from @ericadeahl 

This is what is looks like:

<img width="1264" alt="screen shot 2016-02-25 at 11 06 49 am" src="https://cloud.githubusercontent.com/assets/5249443/13330980/28e696fa-dbb0-11e5-8a34-57b1465dde11.png">

Related: https://github.com/18F/ffd-microsite/issues/1#issuecomment-186387861

maya committed:
Add ratio media query

maya committed:
Remove extra space

maya committed:
Add new website image

ericadeahl commented:
@maya Looks good to me!

rogeruiz commented:
Looks good to me for this specific screen size @maya and @ericadeahl.

I'm seeing issues with the background on some preset devices in Chrome's browser emulation. The issues fall into two categories. **copy on top of background image** and **vertical centering of content in hero**.
- Nexus 5X Landscape ( 731 x 411 )
  - <img width="1388" alt="screen shot 2016-02-25 at 2 30 31 pm" src="https://cloud.githubusercontent.com/assets/706004/13331682/87e1f1f6-dbcc-11e5-80a4-46215e2f50be.png">
- Nexus 6P Landscape ( 773 x 435 )
  - <img width="1388" alt="screen shot 2016-02-25 at 2 26 31 pm" src="https://cloud.githubusercontent.com/assets/706004/13331694/97261228-dbcc-11e5-844d-0593e5e0e362.png">
- Galaxy S5 Landscape ( 640 x 360 )
  - <img width="1388" alt="screen shot 2016-02-25 at 2 29 26 pm" src="https://cloud.githubusercontent.com/assets/706004/13331717/b5b5731e-dbcc-11e5-83be-228b59d55d2c.png">
- iPhone 4 Landscape ( 480 x 320 )
  -  <img width="1388" alt="screen shot 2016-02-25 at 2 28 52 pm" src="https://cloud.githubusercontent.com/assets/706004/13331726/c05d495e-dbcc-11e5-9dd2-f4221e91e4dc.png">
- iPhone 5 Landscape ( 568 x 320 )
  - <img width="1388" alt="screen shot 2016-02-25 at 2 23 33 pm" src="https://cloud.githubusercontent.com/assets/706004/13331732/cc7cd614-dbcc-11e5-85d4-4373ed6b3466.png">

For the **vertical centering of content in hero** issue, I think tweaking the `padding-top` from the default of `6rem` to `3rem` _or something along those lines_ we can get the content to center visually a bit better. As for the other issue **copy on top of background image**, I defer to @ericadeahl for advice on how to better handle that design. i.e. remove the image background?

maya commented:
@rogeruiz thanks, we aren't vertically centering the text but adjusting the margins is a good idea.

ericadeahl commented:
@rogeruiz @maya Agreed that for all these mobile landscape sizes we should remove the background image and try to vertically center that block of header text.

rogeruiz commented:
Cool thanks @ericadeahl. Looks like the `min-height` might be somewhere around `435px` @maya

rogeruiz commented on assets/styles/homepage.scss:
```diff
@@ -64,6 +64,10 @@ p {
+    background-image: url('../images/houses-wide.svg');
+  }
+
   @media (max-height: 400px) {
```
This should be at least `435` in order to make the background disappear a bit earlier.  @maya

rogeruiz commented on assets/styles/homepage.scss:
```diff
@@ -64,6 +64,10 @@ p {
+  }
+
   @media (max-height: 400px) {
     background-image: none;
```
after this line, a `padding-top` smaller than `6em` would have the text visually center a bit better. @maya

maya committed:
Update for landscape sizes

maya commented:
@rogeruiz pushed up an update. Let me know if that works. I changed the min height from 400px to 435px and updated margins.

iPhone 5S
<img width="722" alt="screen shot 2016-02-25 at 12 49 05 pm" src="https://cloud.githubusercontent.com/assets/5249443/13333931/36bb47b8-dbbe-11e5-8b69-8bf0deaa0122.png">

iPhone 6
<img width="823" alt="screen shot 2016-02-25 at 12 40 06 pm" src="https://cloud.githubusercontent.com/assets/5249443/13333933/36bcf900-dbbe-11e5-953f-8d2529b927cc.png">

Galaxy S6
<img width="809" alt="screen shot 2016-02-25 at 1 09 16 pm" src="https://cloud.githubusercontent.com/assets/5249443/13334517/19d363da-dbc1-11e5-9edf-5eb30d0e16c2.png">

iPad mini
<img width="813" alt="screen shot 2016-02-25 at 12 47 12 pm" src="https://cloud.githubusercontent.com/assets/5249443/13333932/36bcb300-dbbe-11e5-9efa-1c43f9dcdb08.png">

rogeruiz commented:
Looks good to me! @ericadeahl can you take a look at @maya's screenshots and let me know if this is good design-wise? Thanks!

ericadeahl commented:
@maya @rogeruiz Looks good! The iPhone 6 one is still a little awkward - it would look better with larger text and less empty space below - but it's not a big deal. We can leave it alone if changing it would mess up the others. Otherwise :+1:

maya commented:
@ericadeahl ya there'll probably be some edge cases like that one, but I think going smaller text and margins will allow us to cover our bases better.

ericadeahl commented:
@maya Sounds good!
